### ISSUE TITLE
Fix code style errors

### ISSUE THREAD

**Issue by xow** [xow/moodle-mod_quizgame#21]
Both moodlecheck and codechecker

**Comment by xow** [comment]
Originally reported in #19

**Comment by xow** [comment]
```
Registering sniffs in the moodle standard... DONE (66 sniffs registered)
Creating file list... DONE (15 files in queue)
Changing into directory /home/vagrant/moodles/project_week/moodle/mod/quizgame
Processing view.php [PHP => 542 tokens in 73 lines]... DONE in 23ms (0 errors, 0 warnings)
Changing into directory /home/vagrant/moodles/project_week/moodle/mod/quizgame/lang/en
Processing quizgame.php [PHP => 277 tokens in 58 lines]... DONE in 14ms (0 errors, 0 warnings)
Changing into directory /home/vagrant/moodles/project_week/moodle/mod/quizgame
Processing locallib.php [PHP => 768 tokens in 113 lines]... DONE in 30ms (0 errors, 0 warnings)
Processing index.php [PHP => 717 tokens in 85 lines]... DONE in 29ms (0 errors, 0 warnings)
Changing into directory /home/vagrant/moodles/project_week/moodle/mod/quizgame/db
Processing access.php [PHP => 321 tokens in 74 lines]... DONE in 11ms (0 errors, 0 warnings)
Processing log.php [PHP => 225 tokens in 38 lines]... DONE in 11ms (0 errors, 0 warnings)
Processing uninstall.php [PHP => 87 tokens in 32 lines]... DONE in 7ms (0 errors, 0 warnings)
Processing install.php [PHP => 125 tokens in 42 lines]... DONE in 7ms (0 errors, 0 warnings)
Processing upgrade.php [PHP => 402 tokens in 77 lines]... DONE in 18ms (0 errors, 0 warnings)
Changing into directory /home/vagrant/moodles/project_week/moodle/mod/quizgame
Processing styles.css [CSS => 62 tokens in 12 lines]... DONE in 5ms (0 errors, 0 warnings)
Processing grade.php [PHP => 127 tokens in 34 lines]... DONE in 9ms (0 errors, 0 warnings)
Processing mod_form.php [PHP => 490 tokens in 80 lines]... DONE in 20ms (0 errors, 0 warnings)
Processing version.php [PHP => 137 tokens in 37 lines]... DONE in 9ms (0 errors, 0 warnings)
Processing lib.php [PHP => 2189 tokens in 390 lines]... DONE in 66ms (0 errors, 0 warnings)
Processing quizgame.js [JS => 9057 tokens in 843 lines]... DONE in 378ms (0 errors, 0 warnings)
<?xml version="1.0" encoding="UTF-8"?>
<checkstyle version="1.3.2">
<file name="mod/quizgame/view.php">
</file>
<file name="mod/quizgame/lang/en/quizgame.php">
</file>
<file name="mod/quizgame/locallib.php">
</file>
<file name="mod/quizgame/index.php">
</file>
<file name="mod/quizgame/db/access.php">
</file>
<file name="mod/quizgame/db/log.php">
</file>
<file name="mod/quizgame/db/uninstall.php">
</file>
<file name="mod/quizgame/db/install.php">
</file>
<file name="mod/quizgame/db/upgrade.php">
</file>
<file name="mod/quizgame/grade.php">
</file>
<file name="mod/quizgame/mod_form.php">
</file>
<file name="mod/quizgame/version.php">
</file>
<file name="mod/quizgame/lib.php">
</file>
</checkstyle>
```

**Comment by xow** [comment]
Fixed in 55e1478 and 414cc96
